grouper/models.py

```diff
@@ -631,6 +635,59 @@ def my_open_audits(self):
         ).all()
 
 
+def _make_secret():
+    return os.urandom(20).encode("hex")
+
+
+class UserToken(Model):
+    """Simple bearer tokens used by third parties to verify user identity"""
+
+    __tablename__ = "user_tokens"
+
+    id = Column(Integer, primary_key=True)
+
+    user_id = Column(Integer, ForeignKey("users.id"))
+    name = Column(String(length=16), nullable=False)
+
+    ctime = Column(DateTime, default=datetime.utcnow, nullable=False)
+    dtime = Column(DateTime, default=None, nullable=True)
```

Comment:
i'm not a huge personal fan of the naming convention that exists but can we make these two more consistent with it? so `ctime` should be `created_on` or `created_at` and 'dtime' should be `disabled_on` or `disabled_at`